analysis_runner/analysisrunner.py

```diff
@@ -33,6 +34,13 @@ def add_analysis_runner_args(parser=None) -> argparse.ArgumentParser:
         parser = argparse.ArgumentParser('analysis-runner subparser')
 
     add_general_args(parser)
+
+    parser.add_argument(
+        '--environment-variables',
+        required=False,
+        help='A dictionary of environment variables',
+    )
```

Comment:
Repeating args with action=append on an argparse 

```suggestion
    action='append'
    )
```